components/brave_rewards/resources/rewards_panel/components/adaptive_captcha_view.tsx

```diff
@@ -75,58 +74,69 @@ export function AdaptiveCaptchaView (props: Props) {
     return () => { window.removeEventListener('message', listener) }
   }, [props.onCaptchaResult])
 
+  function onContactSupport () {
+    window.open('https://support.brave.com/', '_blank')
+    props.onClose()
+  }
+
   function renderCaptcha () {
     return (
-      <styles.frameBox>
-        <iframe
-          ref={iframeRef}
-          allow={iframeAllow.trim().replace(/\n/g, '')}
-          src={props.captchaURL}
-          sandbox='allow-scripts'
-          scrolling='no'
-        />
-      </styles.frameBox>
+      <styles.overlay>
+      <styles.frameRoot>
+      <iframe
+      ref={iframeRef}
```

Comment:
(nit) Indenting

Reply:
Thanks, my editor likes to randomly reformat my code from time to time, sigh - fixed.